src/components/timecard/edit-shift-hours/EditShiftHours.jsx

```diff
@@ -146,11 +146,17 @@ const EditShiftHours = ({
     );
 
     const endTime = formData[`${inputName}-finish-time`] || null;
+    const endDate = `${formData['finishDate-year']}-${formData['finishDate-month']}-${formData['finishDate-day']}`;
```

Comment:
nit: Looks like this var is only used within the the `if` block on line 153. 
Could probs move it there to limit it's scope appropriately.

Reply:
This has been moved, had the same thought

Reply:
noice